components/BookingListPreview.js

```diff
@@ -1,34 +1,72 @@
-import React from 'react';
+import React, { useState, useMemo, useEffect } from 'react';
 import Link from 'next/link';
 import dayjs from 'dayjs';
+import { usePlatform } from '../contexts/platform';
+import { useAuth } from '../contexts/auth';
 
 import { __, priceFormat } from '../utils/helpers';
 
-const ListingListPreview = ({ booking }) => {
-  if (!booking) {
-    return null;
-  }
+const ListingListPreview = ({ status, limit }) => {
 
-  const start = dayjs(booking.get('start'));
-  const end = dayjs(booking.get('end'));
+  const { user } = useAuth();
+  const { platform } = usePlatform();
+  const [page, setPage] = useState(1);
+  const [error, setErrors] = useState(false);
+  const filter = useMemo(() => ({
+    where: { status },
+    limit,
+    page
+  }), [status, limit, page]);
 
+  const bookings = platform.booking.find(filter);
```

Comment:
This component is not the right place for loading bookings - you can add a new `BookingList` component with this and keep `BookingListPreview` just to display each individual listing. Also rename to `BookingListPreview` instead of `ListingListPreview`.